packages/react-storage/src/components/StorageBrowser/views/LocationActionView/DeleteView/DeleteView.tsx

```diff
@@ -53,19 +53,34 @@ export const DeleteView = ({
         onActionStart={onActionStart}
         onActionCancel={onActionCancel}
       >
-        <Exit onClick={onExit} disabled={isProcessing} />
+        <ViewElement className={`${AMPLIFY_CLASS_BASE}__navigation`}>
+          <Exit onClick={onExit} disabled={isProcessing} />
+        </ViewElement>
         <Title />
-        <ViewElement className={`${CLASS_BASE}__table-wrapper`}>
+
+        <ViewElement className={`${AMPLIFY_CLASS_BASE}__table__wrapper`}>
           <DataTableControl
             className={`${CLASS_BASE}__delete-view-data-table`}
           />
         </ViewElement>
-        <ViewElement className={`${CLASS_BASE}__action-footer`}>
+
+        <ViewElement className={`${AMPLIFY_CLASS_BASE}__action__summary`}>
           <StatusDisplayControl
-            className={`${CLASS_BASE}__action-status-display`}
+            className={`${AMPLIFY_CLASS_BASE}__action__status`}
```

Comment:
Think using classnames that deviate from the component names negatively impacts discoverability, is there a strong reason to change this?

Reply:
Maybe `Statuses` since it's a list?